packages/backend/src/open_payments/payment/incoming_remote/service.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  AuthenticatedClient,
+  IncomingPayment as OpenPaymentsIncomingPayment,
+  isNonInteractiveGrant
+} from 'open-payments'
+
+import { AccessType, AccessAction, Grant } from '../../grant/model'
+import { GrantService } from '../../grant/service'
+import { BaseService } from '../../../shared/baseService'
+import { Amount, serializeAmount } from '../../amount'
+
+interface CreateRemoteIncomingPaymentArgs {
+  paymentPointerUrl: string
+  description?: string
+  expiresAt?: Date
+  incomingAmount?: Amount
+  externalRef?: string
+}
+
+export interface RemoteIncomingPaymentService {
+  create(
+    args: CreateRemoteIncomingPaymentArgs
+  ): Promise<OpenPaymentsIncomingPayment>
+}
+
+interface ServiceDependencies extends BaseService {
+  grantService: GrantService
+  openPaymentsUrl: string
+  openPaymentsClient: AuthenticatedClient
+}
+
+export async function createRemoteIncomingPaymentService(
+  deps_: ServiceDependencies
+): Promise<RemoteIncomingPaymentService> {
+  const log = deps_.logger.child({
+    service: 'RemoteIncomingPaymentService'
+  })
+  const deps: ServiceDependencies = {
+    ...deps_,
+    logger: log
+  }
+
+  return {
+    create: (args) => create(deps, args)
+  }
+}
+
+async function create(
+  deps: ServiceDependencies,
+  args: CreateRemoteIncomingPaymentArgs
+): Promise<OpenPaymentsIncomingPayment> {
+  const { paymentPointerUrl } = args
+  const grant = await getGrant(deps, paymentPointerUrl, [
+    AccessAction.Create,
+    AccessAction.ReadAll
```

Comment:
Do we want to have a `GetRemoteIncomingPayment` query in the Admin API in addition to this "Create" one?

Reply:
https://github.com/interledger/rafiki/issues/1005

CC @wilsonianb